Strategy11/formidable-forms
# Add filter for limiting repeater entries in list page

Related issue: https://github.com/Strategy11/formidable-pro/issues/4128

### Test procedure
1. Create a form and add a repeater to it.
2. Add a text input the repeater.
3. Submit many long text values for the input field.
4. View the form entries list and there must be 5 repeated entries per an entry in the list which is the new default.
5. Use the new `frm_repeater_entries_limit` to increase the number of entries displayed in the list.
Example.
`add_filter( 'frm_repeater_entries_limit', function( $limit ) { return 10;} );`
7. Confirm that number of entries is increased accordingly.
